src/inmanta/data.py

```diff
@@ -573,6 +574,8 @@ class Environment(BaseDocument):
                                           doc="Agent interval for autostarted agents in seconds", agent_restart=True),
         SERVER_COMPILE: Setting(name=SERVER_COMPILE, default=True, typ="bool",
                                 validator=convert_boolean, doc="Allow the server to compile the configuration model."),
+        RESOURCE_ACTION_LOGS_RETENTION: Setting(name=RESOURCE_ACTION_LOGS_RETENTION, default=7, typ="int",
+                                                validator=convert_int, doc="Amount of days to retain resource-action logs"),
```

Comment:
This should be "Number of". Days is countable: https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of

Reply:
I approved but this should be changed actually :)